### PR TITLE
Default the Course User name to be the User name.

### PR DESCRIPTION
**Pull request by lowjoel**

### CODE REVIEW

File: spec/models/course_user_spec.rb [hunk 1]
```diff
@@ -15,6 +15,14 @@
 
     it { is_expected.to be_student }
     it { is_expected.not_to be_phantom }
+
+    context 'when a user is provided' do
+      let(:user) { create(:user) }
+      subject { CourseUser.new(user: user) }
```
Comment (allenwq) [inline spec/models/course_user_spec.rb]: How about:

```ruby
subject { CourseUsesr.new(user: user).name }

it { is_expected.to eq(user.name) }
```

File: app/models/course_user.rb [hunk 2]
```diff
@@ -40,4 +42,10 @@ def staff?
   def on_rejected_entry(*)
     destroy
   end
+
+  private
+
+  def set_defaults
```
Comment (allenwq) [inline app/models/course_user.rb]: <code>#:nodoc: </code> ?

File: spec/models/course_user_spec.rb (hunk 1 above)
Comment (lowjoel) [inline spec/models/course_user_spec.rb]: that's not what the context says, the context and subject should be related. we're still talking about the `CourseUser` object.

**Comment by lowjoel** [comment]
Fixed the `:nodoc:` though.

**Pull request merged**